simulariumio/tests/readers/test_custom_trajectory_reader.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+import numpy as np
+
+from simulariumio import Converter, exceptions
+from simulariumio.tests.conftest import three_default_agents
+
+
+@pytest.mark.parametrize(
+    "trajectory, expected_data",
+    [
+        # 3 default agents (radius 5-10) at given positions for 5 frames,
+        # test string for plots
+        (
+            three_default_agents(),
+            {
```

Comment:
FIRST: THANK YOU FOR TESTING

Second:
it may be good to move these large expected data objects into json files and simply read them in before asserting tests. I know others get frustrated when I have huge long parametrized tests because python prints out the entire parametrized set when reporting an error and so the test log becomes a mess.

Reply:
I tried doing this but had issues with parsing, even though it was the same object in the file. Maybe you can help me figure that out, otherwise I'm just going to leave it.